src/pages/wallets/wallets.ts

```diff
@@ -100,8 +104,57 @@ export class WalletsPage {
     }
   }
 
+  private async walletAudienceEvents() {
+    const deviceUUID = this.platformProvider.getDeviceUUID();
+    const hasCreatedWallet = this.persistenceProvider.getHasReportedFirebaseWalletCreateFlag();
```

Comment:
Isn't `await` missing here? I guess all persistent functions are async.

Reply:
Ah, Yes can add that, but while testing it did yield a result